docs/pages/docs/foundation/inkwell.mdx

```diff
@@ -0,0 +1,81 @@
+import { Tabs } from 'nextra/components';
+import { Widget } from "../../../components/widget.tsx";
+import LinkBadge from "../../../components/link-badge/link-badge.tsx";
+import LinkBadgeGroup from "../../../components/link-badge/link-badge-group.tsx";
+
+# Inkwell
+An area that responds to touch.
+
+It is typically used to create other high-level widgets, i.e. [button](../form/button). You should prefer those
+high-level widgets unless you're creating a custom widget.
```

Comment:
Should we make this a callout? I think we can add it to everything inside the foundations folder.